x-pack/plugins/siem/public/components/page/hosts/host_summary/index.tsx

```diff
@@ -99,9 +106,11 @@ export const createDraggable = (
   field: string,
   startDate: number,
   endDate: number,
-  relativeFields: string[]
+  hostName: string
 ) =>
-  summaryValue == null ? (
+  summaryValue == null && hostName != null && ['firstSeen', 'lastSeen'].includes(field) ? (
+    <FirstLastSeenHost hostname={hostName} type={kebabCase(field) as FirstLastSeenHostType} />
+  ) : summaryValue == null ? (
```

Comment:
I think the type signature of this would be better expressed as this which is the reality of what we are seeing:

```ts
export const createDraggable = (
  summaryValue: string | null,
  field: string | FirstLastSeenHostType,
  startDate: number,
  endDate: number,
  hostName: string | null | undefined
) =>
```

The reason why we're not seeing typescript becoming upset with hostName is because we when call the function like so:

```ts
createDraggable(value, field, startDate, endDate, get('host.name', host))
```

The `get('host.name', host)` is returning `any` which turns off type checking. However, at least typing the signatures would help out.

Optionally above this when this `createDraggable` is called if we cast the `get` like so:

```ts
const summaryValue: string | string[] | null = get(field, host);
const hostName: string | null | undefined = get('host.name', host);
...(later on in the code)...
summaryValue.map((value: string) =>
   createDraggable(value, field, startDate, endDate, hostName)
)
...
            ) : (
              createDraggable(summaryValue, field, startDate, endDate, hostName)
            )}
```
That would cause TypeScript to give errors about hostName.